_posts/2022/2022-10-31-use-zero-for-integer-float-comparison.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Use zero? for comparison of numerics like Integer, Float and BigDecimal"
+description: "Other languages don't have this"
+layout: article
+category: ruby
+image:
+  base: "2022/use-zero-for-integer-float-comparison"
+  alt: "A zero on a sign on the wall"
+  credit: "Bernard Hermant"
+  source: "https://unsplash.com/photos/rfpSOlH1JlQ"
+---
+
+Ruby often takes a more “readable” approach to its syntax and standard library of included methods than other languages.
```

Comment:
Compared to other languages, Ruby prioritises readability when it comes to it syntax and standard library ?

Reply:
```suggestion
Compared to other similar language Ruby often prioritises readability and joy when it come to its syntax and methods in its standard library.
```